models/issue_label.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 // LabelColorPattern is a regexp witch can validate LabelColor
-var LabelColorPattern = regexp.MustCompile("^#[0-9a-fA-F]{6}$")
+var LabelColorPattern = regexp.MustCompile("^#?(?:[0-9a-fA-F]{6}|[0-9a-fA-F]{3})$")
```

Comment:
I'm personally not sure about about making the `#` optional, as that would imply you don't need to use hex colors, whereby the only valid input are hex colors(you can see it as some kind of guard)

Reply:
Simplify it a bit:
```suggestion
var LabelColorPattern = regexp.MustCompile("^#?[0-9a-fA-F]{3,6}$")
```

Reply:
I was under the impression that `{3,6}` meant between 3 and 6 characters, as opposed to 3 or 6 characters. That's why I split it up.

Reply:
In terms of disallowing the `#`, my main justification is that I can't think of a single named colour that only uses the characters `a-f`, and hence the user would get a validation error anyway. Main difference is if they forget a `#` when they copy it just accepts it anyway.

Reply:
Woops, nvm that.